src/main/java/com/flytrap/rssreader/presentation/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,40 @@
+package com.flytrap.rssreader.presentation.filter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.flytrap.rssreader.domain.member.Member;
+import com.flytrap.rssreader.infrastructure.properties.AuthProperties;
+import com.flytrap.rssreader.presentation.dto.SessionMember;
+import com.flytrap.rssreader.presentation.resolver.AuthorizationContext;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+import java.io.IOException;
+import javax.security.sasl.AuthenticationException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Slf4j
+@Component
+@Order(2)
+@RequiredArgsConstructor
+public class AuthenticationFilter extends OncePerRequestFilter { //TODO 통합테스트 해보면 좋겠습니다.
+
+    private final ObjectMapper objectMapper;
+    private final AuthProperties authProperties;
+    private final AuthorizationContext context;
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws IOException, ServletException {
+        HttpSession session = request.getSession();
+
+        Object attribute = session.getAttribute(authProperties.sessionId());
```

Comment:
```suggestion
        SessionMember attribute = (SessionMember) session.getAttribute(authProperties.sessionId());
```
타입을 명시해 주면 코드를 더 이해하기 쉬운 것 같아요. 세션에서 가져오는게 뭔지 바로 알 수 있어서요.
근데 최신 언어들 보면 아닌가 싶기도 하고 그렇네요